### PR TITLE
drop support to python <3.5.3

### PR DESCRIPTION
**Pull request by jairhenrique**

### CODE REVIEW

File: aiohttp/helpers.py [hunk 1]
```diff
@@ -219,14 +215,8 @@ def proxies_from_env():
...
-if PY_352:
-    def create_future(loop):
-        return loop.create_future()
-else:
-    def create_future(loop):  # pragma: no cover
-        """Compatibility wrapper for the loop.create_future() call introduced in
-        3.5.2."""
-        return asyncio.Future(loop=loop)
+def create_future(loop):
+    return loop.create_future()
```
Comment (hellysmile) [inline aiohttp/helpers.py]: You can remove this function at all and just use ‘loop.create_future’ everywhere

Comment (jairhenrique) [reply aiohttp/helpers.py]: :+1:

File: .travis.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ sudo: false
 language: python
 
 python:
-- 3.5.2
+- 3.5.3
```
Comment (asvetlov) [inline .travis.yml]: 3.5.2 was here to emphasize the difference between 3.5.2 and 3.5.3
Let's just drop redundant travis job. I could help to reduce travis build time.

Comment (jairhenrique) [reply .travis.yml]: @asvetlov removed.

**Pull request merged**